src/toxext.h

```diff
@@ -66,6 +72,12 @@ typedef void (*toxext_recv_callback)(struct ToxExtExtension *extension,
  * You might think that there should be some way for the extension to version
  * itself and reject the negotiation request, but I argue that on new versions
  * the UUID should just be updated
+ *
+ *  @param friend_id The Tox defined session friend ID of the sender.
+ *  @param compatible ***does not compatible just mean they don't have the extension, or could they somehow have an
+ // incompatible version of the extension with the same UUID?***
```

Comment:
This is supposed to mean that they have the extension IIRC. I think compatible is false if negotiation fails or it times out